Dockerfile

```
@@ -22,7 +22,10 @@ RUN if [ "$WIN_BUILD" = "true" ] ; then apt-get -y install mingw-w64; fi
 RUN pip3 install --no-cache-dir -r requirements.txt
 
 # Set up config file and disable atomic by default
-RUN grep -v "\- atomic" conf/default.yml > conf/local.yml
+RUN if [ -f "conf/local.yml" ]; then sed -i '/\- atomic/d' conf/local.yml;                                             \
+    else sed -i '/\- atomic/d' conf/default.yml;                                                                       \
+    python3 -c "import app; import app.utility.config_generator; app.utility.config_generator.ensure_local_config();"; \
+    fi
```

I tried it out but it wasn't removing the `- atomic` line. Not sure why, but I simplified this section as follows and it worked:

```suggestion
RUN python3 -c "import app; import app.utility.config_generator; app.utility.config_generator.ensure_local_config();"; \
    sed -i '/\- atomic/d' conf/local.yml;
```

---

ah that's odd - updated the dockerfile with this change